WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/AddProductFeatures/AddProductFeaturesView.swift

```diff
@@ -0,0 +1,137 @@
+import SwiftUI
+
+/// View for adding product features in the product creation with AI flow.
+///
+struct AddProductFeaturesView: View {
+    @FocusState private var editorIsFocused: Bool
+    @ObservedObject private var viewModel: AddProductFeaturesViewModel
+
+    init(viewModel: AddProductFeaturesViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: Layout.blockBottomPadding) {
```

Comment:
Nit: What do you think about renaming `Layout.blockBottomPadding` into something like `Layout.blockVerticalSpacing` for clarity?

Reply:
Updated in e012db0e721638094480c9ef0606f8b39f475c68.